install_files/ansible-base/roles/tails-config/tasks/migrate_existing_config.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Between 0.3.12 and 0.4, the "Document Interface" was renamed to
+# "Journalist Interface". Since multiple scripts expect to readfrom this file,
+# we'll rename the old file to handle the transition gracefully.
+- name: Migrate Document Interface ATHS file (upgrade only).
+  command: >-
+    mv --no-clobber
```

Comment:
Simply an abundance of caution... if the new file exists for some reason, I didn't want to lose the info in there. But I guess that's unreasonable given the deployment plans: we can confidently `mv` and clobber.